skunkworks_crow/build.gradle

```diff
@@ -70,4 +70,8 @@ dependencies {
 
     //MPAndroidChart
     implementation 'com.github.PhilJay:MPAndroidChart:v3.0.3'
+
+    // Permissions Dispatcher
+    implementation "org.permissionsdispatcher:permissionsdispatcher:4.5.0"
```

Comment:
Also add about this library in our `Open source license` section.

Reply:
Oooh yes, thanks for reminding, wait a minute.

Reply:
Updated along with `Robolectric` and `Mockito`.